Orange/widgets/evaluate/owpredictions.py

```diff
@@ -296,30 +297,36 @@ def _set_errors(self):
         else:
             self.Warning.wrong_targets.clear()
 
-    def _update_info(self):
+    def _set_input_summary(self):
         if not self.data and not self.predictors:
             self.info.set_input_summary(self.info.NoInput)
             return
 
-        summary = str(len(self.data)) if self.data else "0"
+        summary = len(self.data) if self.data else "0"
         details = self._get_details()
         self.info.set_input_summary(summary, details)
 
     def _get_details(self):
+        details = format_summary_details(self.data) if self.data else \
+            "No data on input"
```

Comment:
This is inconsistent with other widget that accept multiple inputs.
I propose:


Data:
No data on input.
- - -
Model:
No model on input.